pkg/internal/resources/fakeoperator/fakeoperator.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2019 The KubeCarrier Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fakeoperator
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/kustomize/v3/pkg/image"
+	"sigs.k8s.io/kustomize/v3/pkg/types"
+
+	"github.com/kubermatic/kubecarrier/pkg/internal/kustomize"
+	"github.com/kubermatic/kubecarrier/pkg/internal/version"
+)
+
+// Config holds the config information to generate the kubecarrier operator setup.
+type Config struct {
+	// Namespace the e2e operator should be deployed into.
+	Namespace string
+	// Name of this e2e operator object
+	Name string
```

Comment:
This field is used only once, in the `_test.go` file. If it's not important enough you could delete this field. Or maybe you forgot using it somewhere

Reply:
removed